src/ckeditor/ckeditor.component.spec.ts

```diff
@@ -51,6 +51,14 @@ describe( 'CKEditorComponent', () => {
 					} );
 				} );
 
+				it( 'should have editorUrl pointing to 4.13.0 version', () => {
```

Comment:
Since the CDN version will be updated with each release we will have to keep this test in sync. OTOH I don't see any particular reason that it is bad idea to have such test, so let's leave it and see how it works.

Reply:
Well, if there is any way to get the latest version of editor we could turn this into a variable instead of hardcoding, but I don't have any non-hacky idea for this right now 🤔